### PR TITLE
Issue #7419 fixed

### PR DESCRIPTION
**Pull request by vijay-webkul**
## Issue Reference
#7419

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
The **start** and **end** time is **not visible.** please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/642a862e2a666707c408fe0e)

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
This is **working fine.**

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/CartRule/src/Repositories/CartRuleRepository.php [hunk 1]
```diff
@@ -56,9 +56,9 @@ public function model(): string
      */
     public function create(array $data)
     {
-        $data['starts_from'] = $data['starts_from'] ?? null;
+        $data['starts_from'] = $data['starts_from'] ? $data['starts_from'] : null;
 
-        $data['ends_till'] = $data['ends_till'] ?? null;
+        $data['ends_till'] = $data['ends_till'] ? $data['ends_till'] : null;
```
Comment (devansh-webkul) [inline packages/Webkul/CartRule/src/Repositories/CartRuleRepository.php:61]: No need to do these changes.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
Resolved the conflicts.

**Pull request closed without merging**